docs/pages/docs.md

```diff
@@ -5,23 +5,69 @@ layout: default
+| [pdoc]    | Auto-generates API documentation from docstrings, beginner friendly but with less of a plugin ecosystem than others.                                            | 🟠  |
+
+<details markdown="block">
+<summary>More details about Sphinx</summary>
+
+We marginally recommend [MkDocs] over [Sphinx] due to it's ease of use,
+preference for Markdown, and more support for a variety of docstring styles.
+
+However the [Sphinx] tool has long been the de-facto standard in the scientific Python ecosystem. It is widely
+used, customisable, and well tested. If you need a [Sphinx
+extension](#sphinx-extensions) that does not have an equivalent [MkDocs
+plugin](https://github.com/mkdocs/catalog), or if you are part of a community
+that heavily uses [Sphinx] then we recommend you use that
+instead.
+
+### See also
+
+- Our internal discussions about which to recommend ([#16](https://github.com/UCL-ARC/python-tooling/issues/16) and [#187](https://github.com/UCL-ARC/python-tooling/issues/187)).
+- [An interesting related discussion](https://github.com/encode/httpx/discussions/1220).
+
 </details>
 
+<!-- URLS used above -->
+
+[MkDocs]: https://www.mkdocs.org/
+[Sphinx]: https://www.sphinx-doc.org/en/master/
+[gitbook]: https://www.gitbook.com/
+[pdoc]: https://pdoc.dev/
+
+## MkDocs plugins
+
+| name                                                          | short description                            | 🚦  |
+| ------------------------------------------------------------- | -------------------------------------------- | :-: |
+| [mkdocstrings-python](https://mkdocstrings.github.io/python/) | Automatically generates api reference pages. | 🟢  |
+
 ## Sphinx extensions
 
-| Name                                                                 | Short description                                               | 🚦  |
-| -------------------------------------------------------------------- | --------------------------------------------------------------- | :-: |
-| [sphinx-gallery](https://sphinx-gallery.github.io/stable/index.html) | Builds an HTML gallery of examples from a set of Python scripts | 🟢  |
-| [sphinx-autoapi](https://sphinx-autoapi.readthedocs.io/en/stable/)   | Automatically generates API reference pages.                    | 🟢  |
+| name                                                                 | short description                                                | 🚦  |
+| -------------------------------------------------------------------- | ---------------------------------------------------------------- | :-: |
+| [sphinx-gallery](https://sphinx-gallery.github.io/stable/index.html) | Builds an html gallery of examples from a set of python scripts. | 🟢  |
+| [sphinx-autoapi](https://sphinx-autoapi.readthedocs.io/en/stable/)   | Automatically generates api reference pages.                     | 🟢  |
```

Comment:
```suggestion
| [sphinx-autoapi](https://sphinx-autoapi.readthedocs.io/en/stable/)   | Automatically generates API reference pages.                     | 🟢  |
```

Reply:
Huh, something's gone wrong with capitalisation. Sorry for the fail/noise 🙃 
```
| name                                                                 | short description                                                | 🚦  |
```